app/controllers/meetings_controller.rb

```diff
@@ -259,29 +210,48 @@ def destroy
   private
 
   # Use callbacks to share common setup or constraints between actions.
-  # rubocop:disable RescueStandardError
   def set_meeting
     @meeting = Meeting.friendly.find(params[:id])
-  rescue
+  rescue StandardError
```

Comment:
are we rescuing that the record is not found? Can we rescue that error instead so we're not potentially swallowing other errors?

Reply:
Honestly, I am not sure what we are rescuing here. It was a general `rescue` before so I changed it to `StandardError` which is no better but more of a preference issue. Maybe @juliamji could provide us with some answers

Reply:
I think you're referring to the wrong person... but yeah looks like the record not being found is being rescued.

Reply:
oops, my bad. Okay in that case we can rescue from `RecordNotFound` instead